src/DataProtection/Cryptography.Internal/src/CryptoUtil.cs

```diff
@@ -76,12 +76,22 @@ public static Exception Fail(string message)
 #endif
         public static bool TimeConstantBuffersAreEqual(byte* bufA, byte* bufB, uint count)
         {
+#if NETCOREAPP
```

Comment:
Should we also update the overload that takes a byte array to use this new API?

Reply:
Ah yeah, I missed that good catch!